[documentation] (#338)
DataDog/dd-sdk-android: docs/rum_collection.md

  
  **Note**: For `ActivityViewTrackingStrategy`, `FragmentViewTrackingStrategy`, or `MixedViewTrackingStrategy` you can filter which `Fragment` or `Activity` is tracked as a RUM View by providing a `ComponentPredicate` implementation in the constructor.
  

>   **Note**: By default RUM Monitor for View tracking runs in manual mode so if you decide not to provide a view tracking strategy you will have to manually send the views by calling the start/stop View API methods yourself.

Reviewer: Could we maybe change the "RUM Monitor for View" to "RUM view tracking strategy runs...". RUM Monitors is already a product and that could create confusion, what do you think?

Also about the meaning of that note: if it runs in manual mode, I would have to manually send the views anyway?

Reviewer 2: Indeed, we could instead say: 
> By default, the library won't track any View. If you decide not to provide a view tracking strategy you will have to manually send the views by calling the `startView`/`stopView` methods yourself.

Author: ```suggestion
  **Note**: By default, the library won't track any view. If you decide not to provide a view tracking strategy you will have to manually send the views by calling the `startView` and `stopView` methods yourself.
```
